packages/apps/src/Apps.tsx

```diff
@@ -26,7 +27,9 @@ interface SidebarState {
 
 function WarmUp (): React.ReactElement {
   const { api, isApiReady } = useApi();
-  const fees = useCall<any>(isApiReady ? api.derive.balances?.fees : undefined, []);
+  // Commented for easy tracking while rebasing'
```

Comment:
Lets use it as a pattern for this kind of changes, then we could easily search all places we tweaked :)